tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "jsx": "react",
   "compilerOptions": {},
```

Comment:
please update compilerOptions, IDEs will need it to properly handle jsx files:
"compilerOptions": {
    "jsx": "react"
},